processing/src/main/java/org/apache/druid/query/aggregation/AggregatorUtil.java

```diff
@@ -166,6 +166,10 @@ public class AggregatorUtil
    */
   public static List<PostAggregator> pruneDependentPostAgg(List<PostAggregator> postAggregatorList, String postAggName)
```

Comment:
the documentation and var naming in this function is a bit confusing - since it can be used to prune postAggs not needed for a topn metric. Do we want to think that even topn metrics are sort of postAggs?

Reply:
Yeah, I think I understand where you are coming from. I've updated the docs and the name of the second argument to hopefully make it more clear what this is doing.